### PR TITLE
feat: refactor part of utils

### PR DESCRIPTION
**Pull request by SNCubes**

### CODE REVIEW

File: packages/snjs/lib/utils.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ export function isWebCryptoAvailable() {
 /**
  * Whether we are in React Native app
  */
-export function isReactNativeEnvironment() {
+export function isReactNativeEnvironment() : boolean {
```
Comment (mobitar) [inline packages/snjs/lib/utils.ts:39]: Incorrect spacing, should be using auto-linter. It's option + shift + f on Mac also to perform auto-format.

File: packages/snjs/lib/utils.ts [hunk 2]
```diff
@@ -75,7 +74,7 @@ export function concatArrays(...args: any[]) {
 /**
  * @returns Whether the value is a function or object
  */
-export function isObject(value: any) {
+export function isObject<T>(value: T|null): boolean {
```
Comment (mobitar) [inline packages/snjs/lib/utils.ts:77]: Incorrect spacing in T|null

Comment (SNCubes) [reply packages/snjs/lib/utils.ts:77]: Thanks! I had to turn it off because the file was getting distracting. I'll turn it back on and run it.

**Comment by mobitar** [comment]
Hmm not sure why it's counting my commits as changes. This would make it impossible to review your changes in isolation. If we can't figure that out I would recommend creating a new PR by cherry-picking your commits.

**Comment by baptiste-grob** [comment]
Yeah you'll want to merge `master` into your branch :)

**Pull request closed without merging**